caffe2/onnx/helper.h

```diff
@@ -71,6 +72,25 @@ inline AttributeProto MakeAttribute(
   return attr;
 }
 
+inline AttributeProto MakeAttribute(const std::string& name, TensorProto& val) {
+  AttributeProto attr;
+  attr.set_name(name);
+  std::string val_buf;
+  *(attr.mutable_t()) = val;
+  attr.set_type(AttributeProto::TENSOR);
+  return attr;
+}
+
+inline TensorProto MakeTensor(const string& name, const std::vector<float>& v) {
+  TensorProto ret;
+  ret.set_name(name);
+  ret.add_dims(v.size());
+  ret.set_data_type(TensorProto::FLOAT);
```

Comment:
Can we make it general? vector template with char, and pass the data_type as parameter?

Reply:
I guess we can, if data_type is passed as parameter.